README.md

```
@@ -66,7 +66,7 @@ No config files. No integration wiring. No per-seat pricing. No data leaving you
 
 ### Mobile (390px viewport)
 | Login | Dashboard | Scan Results |
-|-------|-----------|--------------|
+|-------|-----------|--------------||
 | ![](docs/screenshots/mobile/login.png) | ![](docs/screenshots/mobile/dashboard.png) | ![](docs/screenshots/mobile/scan-results.png) |
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)